### PR TITLE
[CIA-3079] Update application

### PR DESCRIPTION
**Pull request by kml**
[CIA-3079]

1) updated ruby
2) updated gems
3) updated rails `bin/rails app:update`

Test: https://github.com/toptal/testbox/pull/298
Local test: follow https://github.com/toptal/testbox_bookstore_api/blob/7853440d59cb7dba71389a2bda9db8a30ea3d7cb/README.md

[CIA-3079]: https://toptal-core.atlassian.net/browse/CIA-3079?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -9,7 +9,9 @@ default: &default
   username: <%= ENV.fetch('MYSQL_USER') %>
   password: <%= ENV.fetch('MYSQL_PASSWORD') %>
   database: <%= ENV.fetch('MYSQL_DATABASE') %>
-  host: mysql
+  host: <%= ENV.fetch('MYSQL_HOST', 'mysql') %>
+  encoding: utf8mb4
+  charset: utf8mb4
```
Comment (kml) [inline config/database.yml:14]: based on https://qiita.com/naota7118/items/4f8aeacfd93377d858c4
(whatever it descibes :sweat_smile: )

**Comment by kml** [comment]
@toptal-anvil ping reviewers

**Review by sasha370 — APPROVED** [review]
:rocket:  :+1:

**Review by velichkoStoev — APPROVED** [review]

File: bin/rails [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env ruby
 APP_PATH = File.expand_path('../config/application', __dir__)
-require_relative '../config/boot'
-require 'rails/commands'
+require_relative "../config/boot"
```
Comment (dgsuarez) [inline bin/rails:3]: Should we just update the rubocop rule? We always use single quotes...

**Review by dgsuarez — APPROVED** [review]

**Review by jg — APPROVED** [review]

File: bin/rails (hunk 2 above)
Comment (kml) [reply bin/rails:3]: We don't have RuboCop configured in that repository.
Rails, by default, uses double quotes (see https://github.com/rails/rails/pull/38841)

I'd keep it the way `bin/rails app:update` generates it. If we want to introduce RuboCop, we should also add a companion CI check.

Comment (kml) [reply bin/rails:3]: https://toptal-core.atlassian.net/browse/CIA-3110

**Pull request merged**